assets/src/application/ui/layout/entityActionsMenu/EntityActionsMenu.tsx

```diff
@@ -3,16 +3,7 @@ import classNames from 'classnames';
 
 import './style.scss';
 
-export enum EntityActionsMenuLayout {
-	Horizontal = 'horizontal',
-	Vertical = 'vertical',
-}
-
-export interface EntityActionsMenuProps {
-	className?: string;
-	layout?: EntityActionsMenuLayout;
-	menuItems: Array<React.ReactNode>;
-}
+import { EntityActionsMenuProps, EntityActionsMenuLayout } from './types';
```

Comment:
👍 
i think I prefer just moving all types and interfaces into a corresponding types file, at least for folders/components that have any degree of complexity to them